### PR TITLE
[OPERATOR-549] remove pure-telemetry-certs on uninstallation

### PR DESCRIPTION
**Pull request by haibinxie**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

File: drivers/storage/portworx/component/telemetry.go [hunk 1]
```diff
@@ -122,6 +159,10 @@ func (t *telemetry) Delete(cluster *corev1.StorageCluster) error {
 		return err
 	}
 
+	if err := k8sutil.DeleteSecret(t.k8sClient, TelemetryCertName, cluster.Namespace); err != nil {
```
Comment (piyush-nimbalkar) [inline drivers/storage/portworx/component/telemetry.go:162]: We should pass the ownerRef here too.

File: drivers/storage/portworx/component/telemetry.go [hunk 2]
```diff
@@ -104,6 +104,9 @@ func (t *telemetry) Reconcile(cluster *corev1.StorageCluster) error {
 	if err := t.reconcileCCMProxyConfigMap(cluster, ownerRef); err != nil {
 		return err
 	}
+	if err := t.SetTelemetryCertOwnerRef(cluster, ownerRef); err != nil {
```
Comment (piyush-nimbalkar) [inline drivers/storage/portworx/component/telemetry.go:107]: Let's make this a private method (small camel case)

**Review by nikolaypopov — APPROVED** [review]
LGTM

**Review by piyush-nimbalkar — APPROVED** [review]
Let's add unit tests too.

**Comment by haibinxie** [comment]
@piyush-nimbalkar resolved, will merge once the build finishes

**Pull request merged**